signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java

```diff
@@ -333,6 +338,25 @@ public Bitmap getTransparentSignatureBitmap(boolean trimBlankSpace) {
       return Bitmap.createBitmap(mSignatureBitmap, xMin, yMin, xMax - xMin, yMax - yMin);
     }
 
+    private boolean isDoubleClick() {
+        if (mClearOnDoubleClick) {
+            if (mFirstClick != 0 && System.currentTimeMillis() - mFirstClick > 200) {
```

Comment:
Could you extract the millisecond `200`, value into a `private static final int DOUBLE_CLICK_DELAY = 200;`?